common/app/uiComponent/core/RenderingActor.scala

```diff
@@ -0,0 +1,23 @@
+package uiComponent.core
+
+import akka.actor.Actor
+import model.ApplicationContext
+import uiComponent.UIComponent
+import scala.util.Try
+
+case class Rendering(component: UIComponent, ac: ApplicationContext)
+
+case class RenderingException(error: String) extends RuntimeException(error)
+
+class RenderingActor extends Actor with JavascriptRendering {
+
+  override def javascriptFile: String = "ui.bundle.server.js"
```

Comment:
If it does mean smaller components, then its really the naming which needs to change. Right now (subject to change) the `RenderingActor` will only serve a full page. Yes this full page is made up of components, but its completely up to the JS app to figure that all out. 

In other words, the serverside code doesn't "compose" anything at all. It is only aware of the final product. 

Maybe `UIComponent` should be renamed to `pageToRender` or `html` or something more clear.